subsys/nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -135,11 +135,6 @@
 #define CRACEN_SPAKE_DRIVER_ID 62
 #define CRACEN_SRP_DRIVER_ID   63
 
-/* This option is deprecated and unsupported */
```

Comment:
is this set?

Reply:
evidently this define is not set or else we would get an #error :)